wormchain/syncing.md

```diff
@@ -0,0 +1,141 @@
+# Syncing Mainnet Wormchain
+
+# Contents
+* [Sync From Snapshot](#sync-from-snapshot)
+* [Sync Manually](#sync-manually)
+
+# Sync From Snapshot
+
+## Build Latest Wormchain Release
+
+```bash
+# checkout git repository
+git clone https://github.com/wormhole-foundation/wormhole
+
+# checkout latest release (v2.23.0 at time of writing)
+cd wormhole
+git checkout v2.23.0
+
+# build wormchain
+cd wormchain
+make build/wormchaind
+```
+
+## Get a Recent Wormchain Snapshot
+
+Please ask the Wormhole contributors for a recent Wormchain snapshot to use.
+
+You can also use daily snapshots exported by the CryptoCrew team here: [https://github.com/clemensgg/CryptoCrew-Validators/blob/main/chains/wormchain/service_Node_Snapshot.md](https://github.com/clemensgg/CryptoCrew-Validators/blob/main/chains/wormchain/service_Node_Snapshot.md)
+
+## Download and Clean Snapshot
+
+After you download the snapshot, you'll need to clear the wasm cache:
+
+```bash
+# before starting your node
+rm -r /data/wasm/cache
+```
+
+Now you should be able to successfully start your node!
+
+### Troubleshooting
+
+> We were just able to successfully statesync when retaining the `wasm` folder and deleting the wasm cache, just fyi:
+> 
+> - you need the `wasm` folder from a synced node
+> - then configure statesync + `unsafe-reset-all`
+> - then copy in wasm state `cp -r <your-wasm-folder-location> $DAEMON_HOME/data`
+> - delete wasm cache `rm -r $DAEMON_HOME/data/wasm/cache`
+> - then start the node & statesync
+
+# Sync Manually
+
+## Build Wormchain Versions
+
+```bash
+cd wormhole/wormchain
+
+# v2.14.7
+git checkout v2.14.7
+make build/wormchaind
+mv build/wormchaind build/wormchaind-v2.14.7
+
+# v2.14.9.1
+git checkout v2.14.9.1
+make build/wormchaind
+mv build/wormchaind build/wormchaind-v2.14.9.1
+
+# v2.14.9.6
+git checkout v2.14.9.6
+make build/wormchaind
+mv build/wormchaind build/wormchaind-v2.14.9.6
+
+# v2.18.1
+git checkout v2.18.1
+make build/wormchaind
+mv build/wormchaind build/wormchaind-v2.18.1
+
+# v2.23.0
+git checkout v2.23.0
+make build/wormchaind
+mv build/wormchaind build/wormchaind-v2.23.0
+```
```

Comment:
There will be a new step here only if we make another consensus-breaking change to Wormchain. We should be set for now since that isn't planned in the near future.

That said, it'd be useful to have a script. We just haven't compiled and tested one yet.